src/process_user_code.py

```diff
@@ -100,7 +100,7 @@ def handle_user_prints():
 
 # Execute User Code Thread
 def execute_user_code(abs_path_to_code_file):
-    cpx._Express__abs_path_to_code_file = abs_path_to_code_file
```

Comment:
Do we still need this cpx field? (It looks like we use it at `debug_user_code.py`)

Reply:
True! I'll fix that in debug_user_code :)